proposals/csharp-7.0/tuples.md

```diff
@@ -0,0 +1,437 @@
+# Tuples 
+
+This proposal specifies the changes required to the [C# 6.0 (draft) Language specification](../../spec/introduction.md) to support *Tuples* as a new [value type](../../spec/types.md#value-types).
+
+## Changes to [Lexical structure](../../spec/lexical-structure.md)
```

Comment:
💭  I would not expect tuples to fall under Lexical structure.

Reply:
Could you please say why not, and where you think they shoudl go?